vmm/src/cpu.rs

```diff
@@ -44,14 +53,23 @@ use std::os::unix::thread::JoinHandleExt;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::{Arc, Barrier, Mutex};
 use std::{cmp, io, result, thread};
+#[cfg(feature = "guest_debug")]
+use std::io::Write;
+#[cfg(feature = "guest_debug")]
+use std::mem::size_of as size_of;
+#[cfg(feature = "guest_debug")]
+use linux_loader::elf::Elf64_Nhdr;
 use thiserror::Error;
 use vm_device::BusDevice;
 use vm_memory::GuestAddress;
 use vm_memory::GuestMemoryAtomic;
+#[cfg(feature = "guest_debug")]
+use vm_memory::ByteValued;
 use vm_migration::{
     Migratable, MigratableError, Pausable, Snapshot, SnapshotDataSection, Snapshottable,
     Transportable,
 };
+
```

Comment:
Blank line added, please remove it, as this will prevent `cargo fmt` from organizing the list of imports alphabetically.

Reply:
Okay.